plugins/inlinetoolbar/plugin.js

```diff
@@ -143,10 +142,25 @@
 			 * @param {CKEDITOR.ui.button[]/CKEDITOR.ui.richCombo[]} items Array of UI elements objects.
 			 */
 			CKEDITOR.ui.inlineToolbarView.prototype.renderItems = function( items ) {
-				var output = [];
+				var output = [],
+					isStarted = false;
```

Comment:
This `isStarted` implementation is a bit dirty. It could be implemented in a simpler way. I have added a followup pull request with a proposed array-based simplification in #1129.